app/models/services.py

```diff
@@ -31,13 +28,23 @@ def insert(shop_id="", name="", description="", images=None):
             "name": name,
             "slug": slugify(name),
             "description": description,
-            "images": images,
+            "price": price,
+            "images": images or [],
         }
     )
+    return id
 
-    # TODO: upload images into disk
-    pass
 
+def update(id="", shop_id="", name="", description="", price="", images=None):
+    query = tinydb.Query()
+    doc = {
+        "name": name,
+        "slug": slugify(name),
+        "description": description,
+        "price": price,
+        "images": images if images is not None else [],
```

Comment:
for the update, we should only update the provided fields, 

for example, if I want to update the service name but with keeping the images, this method will reset the images to []

Reply:
yup, I'll update it